style.css

```diff
@@ -0,0 +1,451 @@
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.container .lets_contact {
+  width: 147px;
+  height: 24px;
+  margin-top: 90px;
+  padding-left: 12px;
+}
+
+.container .lets_contact .contact {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 11px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.social_media {
+  padding-top: 0%;
+  margin: 0 auto;
+  display: flex;
+}
+
+.social-icons-set {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.social-icon {
+  list-style: none;
+  margin: 10px;
+}
+
+.social-icon a:hover {
+  color: #6070ff;
+}
+
+.social-icon a i {
+  font-size: 20px;
+}
+
+.social-icon + .social-icon {
+  margin-left: 10px;
+}
+
+/* coming soon */
+
+/* Start middl section  */
+
+.work_section {
+  display: grid;
+}
+
+.middle_section {
+  height: 552px;
+  width: 343px;
+  margin: 100px 16px 0 16px;
+  border-radius: 15px;
+  background-color: white;
+  padding-bottom: 20px;
+}
+
+.middle_section div.pic {
+  width: 295px;
+  height: 220px;
+  margin: 20px 23px;
+  gap: 20px;
+}
+
+.middle_section div.name {
+  width: 295px;
+  height: 40px;
+  margin: 5px 20px;
+}
+
+.middle_section div.name h1 {
+  font-weight: 700;
+  font-size: 32px;
+  columns: #172b4d;
+  line-height: 44px;
+}
+
+.middle_section div.descript {
+  width: 295px;
+  height: 30px;
+  margin: 5px 20px;
+}
+
+.middle_section div.descript div.camp {
+  width: 30%;
+  color: #344563;
+}
+
+.middle_section div.descript div.year {
+  width: 70%;
+  color: #7a869a;
+  font-size: 13px;
+}
+
+.middle_section div.descript div.year ul.list {
+  justify-content: start;
+}
+
+.middle_section div.descript div.year ul.list li {
+  margin: 0 15px;
+}
+
+.middle_section div.task {
+  width: 295px;
+  height: 76px;
+  margin: 5px 20px;
+}
+
+.middle_section div.task p {
+  color: #344563;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.middle_section div.web {
+  width: 295px;
+  height: 24px;
+  margin: 0 20px;
+}
+
+.middle_section div.web div.right {
+  width: 80%;
+  justify-content: center;
+}
+
+.middle_section div.web div.right ul {
+  list-style-type: none;
+  justify-content: space-around;
+  display: flex;
+  width: 100%;
+}
+
+.middle_section div.link a {
+  text-decoration: none;
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  padding: 5px 0 0 4px;
+  height: 100%;
+  width: 100%;
+}
+
+section.about div.link a {
+  text-decoration: none;
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  padding: 7px 0 0 13px;
+  height: 100%;
+  width: 100%;
+}
+
+.middle_section div.link a:hover {
+  background-color: #84a7ff;
+  color: white;
+  font-weight: bold;
+  height: 100%;
+  width: 100%;
+}
+
+section.about div.link a:hover {
+  background-color: #84a7ff;
+  color: black;
+  font-weight: bold;
+}
```

Comment:
- [ ] Once again as suggested by the previous reviewer please make sure you follow the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37-2238&t=DNbJLlVbOz4jR15U-0)🎨 for the `:hover` and `:active` button styling as shown below:
      ![firefox_7KIb3eS8J1](https://user-images.githubusercontent.com/104968236/228607443-ba65137b-bf7a-4601-b413-45b7d8d87a67.png)
    - [ ] So when it's on `:hover` state the background should change to `#6070FF` and you should have a **white** color for the text as well. 😉
    - [ ] Also please check the Figma design for the `:active` state as you haven't implemented that as well.